docs/source/api/apollo-gateway.mdx

```diff
@@ -557,3 +566,155 @@ The details of the `fetch` response sent by the subgraph.
 </tr>
 </tbody>
 </table>
+
+## `class IntrospectAndCompose`
+
+The drop-in replacement for `serviceList`. This class is meant to be passed directly to `ApolloGateway`'s `supergraphSdl` constructor option.
```

Comment:
"This class is meant to be pass directly" -> "You pass an instance of this class directly"